math/fgsl/Portfile

```diff
@@ -16,20 +17,33 @@ license             GPL-2
 description         Fortran interface to the GNU scientific library
 long_description    A portable, object-based Fortran interface to the GNU scientific library \
                     (GSL), a collection of numerical routines for scientific computing. \
-                    Version 1.5.x is for use with GSL versions >= 2.6. \
+                    Version 1.6.x is for use with GSL versions >= 2.7. \
                     Source: https://github.com/reinh-bader/fgsl
 
 homepage            https://doku.lrz.de/fgsl-a-fortran-interface-to-the-gnu-scientific-library-10746505.html
 
-checksums           rmd160  fec0e747a13906fdf0a9abdc43a4204769ef4a5d \
-                    sha256  5013b4e000e556daac8b3c83192adfe8f36ffdc91d1d4baf0b1cb3100260e664 \
-                    size    2945012
+# Switch to private repo, not github, 2024 May 26.  See home page.
```

Comment:
I don't understand why this is being switched to a private repository even after looking at the page.

Reply:
> I don't understand why this is being switched to a private repository even after looking at the page.

Good question.

1.  It is the primary download archive advertised by the developers.
2.  The previous setup with Github portgroup and new version number, failed to fetch.

Github has made fetching artifacts unstable and complicated.  After recent experiences, I now favor developer hosted archives over Github, unless specifically directed by the developers.  What do you think?

Reply:
I don't know what the right thing is. @reneeotten might have an opinion.

Reply:
IMHO this should still just be using the `github` PortGroup and all the useful things it sets automatically. The only thing @Dave-Allured missed here I think is that the files are now prefixed with a `v` and that's why it appears that things are not working correctly. 

You should do this instead and it will all work out: 
```github.setup        reinh-bader fgsl 1.6.0 v```